clarity_ext/service/file_service.py

```diff
@@ -58,6 +58,16 @@ def __init__(self, artifact_service, file_repo, should_cache, os_service, upload
         self.os_service.makedirs(self.temp_path)
         self.os_service.makedirs(self.downloaded_path)
 
+    def parse_xlsx(self, f):
+        from openpyxl import load_workbook
+        import pandas as pd
+        with open(f.name, "rb") as wbook:
+            writer = pd.ExcelWriter(wbook, engine='openpyxl')
+            workbook = load_workbook(wbook, data_only=True)
+
+            writer.book = workbook
+            return writer.book
+
```

Comment:
I tried this locally but I get an `AttributeError: can't set attribute` when reaching `writer.book = workbook` :confused: 

Is `pd.ExcelWriter` necessary here or can you return `workbook` immediately?